SPECS/icingaweb2-module-director.spec

```diff
@@ -189,6 +189,12 @@ done
 %attr(4755, -, -) %{_bindir}/sas2ircu
 
 %changelog
+* Thu Jan 13 2022 Nino Ryschawy <nino.ryschawy@nmsprime.com> - 1.8.1-4
+- fix: Add dependency rh-php-73-process for director
```

Comment:
why not the php80-process dependency? As we use php80 and the director is compatible to php 8 it should also work.

Reply:
is resolved, as it would require changing SPEC of icingaweb2 and this is just too much hassle